.travis.yml

```diff
@@ -31,3 +31,5 @@ after_success:
   - bash scripts/prep-key.sh
   - 'if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then bash scripts/upload-apk.sh; fi'
 
+before_install:
+  - yes | sdkmanager "platforms;android-28"
```

Comment:
Instead try ${ANDROID_HOME}tools/bin/sdkmanager and see

Reply:
the issue is with downloading the cache